### PR TITLE
Copy project files from the last Todo-List project.

### PR DESCRIPTION
**Pull request by adel-gu**
In this PR I copied the last Todo List project to this current repo.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-  const element = document.createElement('div');
-
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-
-  return element;
-}
-
-document.body.appendChild(component());
+// Select DOM Elements
+const form = document.forms[0];
```
Comment (adel-gu) [inline src/index.js:5]: Please change this.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+const taskInput = form.task;
+const todoList = document.querySelector('.todo_body_list');
+const clearBtn = document.querySelector('.clear');
+
+base.createTask(taskInput, todoList, form);
+base.deleteTask(todoList);
+base.editTask(todoList);
+base.loadTasks(todoList);
+base.checkTask(todoList);
```
Comment (adel-gu) [inline src/index.js:14]: this is to.

**Review by adel-gu — COMMENTED** [review]
...

File: src/modules/utils/read.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+export default class Read {
+  // Get data from local storage
+  static getTasks = () => {
+    let tasks = [];
+    if (localStorage.getItem('Tasks')) {
+      tasks = JSON.parse(localStorage.getItem('Tasks'));
+    }
+
```
Comment (UsirusenIkon) [inline src/modules/utils/read.js]: [OPTIONAL] 
remove empty space in function scope

File: src/modules/base.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
+import Create from './utils/create';
+import Delete from './utils/delete';
+import Edit from './utils/edit';
+import Check from './utils/check';
```
Comment (UsirusenIkon) [inline src/modules/base.js:4]: - [OPTIONAL] I suggest you put all methods under one class and also initiate the constructor function under that class. I suggest this is better for readability and organization.

**Review by UsirusenIkon — COMMENTED** [review]
### Hi @adel-gu , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code
:heavy_check_mark: Good descriptive comments
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/modules/base.js (hunk 4 above)
Comment (adel-gu) [reply src/modules/base.js:4]: Hi! @UsirusenIkon  👋

Thank you very much for the review 👏👏👏.

File: src/modules/utils/read.js (hunk 3 above)
Comment (adel-gu) [reply src/modules/utils/read.js]: **Hi @UsirusenIkon👋**, 

I have no reason to let the empty space only for **facilitating the code readability** or at least this is what I thought😁, so Thanks for mentioning this however, can you please provide a best practices context for doing so?!.

**Review by UsirusenIkon — APPROVED** [review]
Hi @adel-gu     , 

Your project is complete and design is beautiful 👍 !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**